bin/clone_aomp.sh

```diff
@@ -303,6 +303,38 @@ if [[ "$AOMP_VERSION" == "13.1" ]] || [[ $AOMP_MAJOR_VERSION -gt 13 ]] ; then
          ln -sf hsa-runtime src
       fi
    fi
+
+if [ "$AOMP_MONO_CLONE" != "" ]; then
+   # comgr now lives in llvm-project/amd/comg
+   # And needs to be found in rocm-compilersupport/lib
+   if [ -d $AOMP_REPOS/rocm-compilersupport ] ; then
+      if [ ! -L $AOMP_REPOS/rocm-compilersupport/lib/comgr ] ; then
+         echo "Fixing rocm-compilersupport with correct link to ../../llvm-project/amd/comgr"
+         echo rm -rf $AOMP_REPOS/rocm-compilersupport/lib
+         rm -rf $AOMP_REPOS/rocm-compilersupport/lib
```

Comment:
Here and below: Instead of echoing the command and then doing it, should we set a variable to the command and then print it and run it, so they never diverge?

```
CmdVar=rm -rf bla
echo $CmdVar
$CmdVar
```